templates/stats/issues.tpl

```diff
@@ -172,6 +172,19 @@
 										</pkp-button>
 									</template>
 								</action-panel>
+								<action-panel class="pkpStats__reportAction">
```

Comment:
I also wasn't seeing this action panel show up in the UI anywhere.

Reply:
![Screenshot from 2022-11-04 14-40-08](https://user-images.githubusercontent.com/378421/199986586-60d85515-0024-43b4-abb6-fff17bc820d6.png)

Reply:
Hi @ewhanson, this should be the `Timeline` part in the screenshot.
This modal opens when you click on the Download button on the Statistics > Issue i.e. Journal/Press/Server page.
Do you see it?

Reply:
Yes, I see it now. Somehow missed the modal. All good!